editor/src/components/canvas/canvas-strategies/strategies/basic-resize-strategy.tsx

```diff
@@ -66,20 +66,19 @@ export function basicResizeStrategy(
   if (selectedElements.length !== 1 || !honoursPropsSize(canvasState, selectedElements[0])) {
     return null
   }
+
+  const selectedElement = selectedElements[0]
+
   const metadata = MetadataUtils.findElementByElementPath(
     canvasState.startingMetadata,
-    selectedElements[0],
+    selectedElement,
   )
   const elementDimensionsProps = metadata != null ? getElementDimensions(metadata) : null
   const elementParentBounds = metadata?.specialSizeMeasurements.immediateParentBounds ?? null
 
-  const elementDimensions =
-    elementDimensionsProps == null
-      ? null
-      : {
-          width: elementDimensionsProps.width,
-          height: elementDimensionsProps.height,
-        }
+  if (MetadataUtils.isGridCell(canvasState.startingMetadata, selectedElement)) {
+    return null
+  }
```

Comment:
Don't you have to disable the other resize strategies (see https://github.com/concrete-utopia/utopia/blob/master/editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx#L104 )

Reply:
yeah https://github.com/concrete-utopia/utopia/pull/6043/commits/ce28bc784b583ad9d216cc8e8718c4906f71b73e